tests/baselines/reference/objectTypeWithRecursiveWrappedPropertyCheckedNominally.errors.txt

```diff
@@ -15,7 +15,6 @@ tests/cases/conformance/types/typeRelationships/recursiveTypes/objectTypeWithRec
 tests/cases/conformance/types/typeRelationships/recursiveTypes/objectTypeWithRecursiveWrappedPropertyCheckedNominally.ts(42,5): error TS2322: Type 'U' is not assignable to type 'T'.
   'U' is assignable to the constraint of type 'T', but 'T' could be instantiated with a different subtype of constraint 'MyList<number>'.
     Type 'MyList<number>' is not assignable to type 'T'.
-      'MyList<number>' is assignable to the constraint of type 'T', but 'T' could be instantiated with a different subtype of constraint 'MyList<number>'.

```

Comment:
marking this

Reply:
This is similar to the other - `T`'s constraint, `MyList<number>` is instantiated with a `this = T`, which is then enough to make it fail the following assignability check.